Gemfile.lock

```diff
@@ -267,6 +267,7 @@ GEM
       unf_ext
     unf_ext (0.0.8.2)
     unicode-display_width (1.8.0)
+    webrick (1.8.1)
```

Comment:
Whats this do?

Reply:
Not sure, I didnt add it? I just pulled the repo and ran the docker container.

Reply:
Cool